docs/ucr/mobile_ucr_v2_migration_guide.rst

```diff
@@ -0,0 +1,80 @@
+Migrating to Mobile UCR Restore V2
+==================================
+
+Evolution of Mobile UCR Restore Versions
+----------------------------------------
+Historically, there have been three versions for how Mobile UCRs were sent to mobile devices:
+
+- **Version 1.0**: All reports are bundled together into a single fixture during the restore process.
+- **Version 1.5**: Both V1 and V2 are supported together. This is used to migrate from V1 without causing any breaking changes.
+- **Version 2.0**: Enhanced performance where each report has a fixture of its own.
+
+Deprecation of Mobile UCR V1 and V1.5
+-------------------------------------
+Both V1 and V1.5 have been deprecated to address the performance challenges encountered with these versions.
+
+Benefits of Upgrading to Mobile UCR V2
+--------------------------------------
+The introduction of Mobile UCR V2 offers significant performance and efficiency improvements around how UCR data is restored and synced.
+These improvements offer faster report lookups and restores, as well as a new sync delay option for each report that allows administrators to control
+how frequently report data is synced.
+
+Moving to Mobile UCR V2
+-----------------------
+- Applications that do not have any references to UCRs in forms, case lists and filters need only have their
+  Mobile UCR Version updated to V2 and the app will continue working as normal.
+- Applications with references to reports will need to have all manual references to the `commcare:reports`
+  fixture changed before updating to V2. These are most likely to be contained in reports modules,
+  but others may be in case lists.
+
+The above migration steps will need to be carried out for any old app versions that get restored. These restored
+app versions will first need to be migrated to Mobile UCR V2 before they can be released.
+
+Update Mobile UCR Version
+-------------------------
+In the app settings, in the "Advanced Setting" tab, you'll find "Mobile UCR Restore Version".
+Set the version to '2.0' if your application does not have any references to UCRs.
```

Comment:
"if your application does not have any references to UCRs."
And for those that have references?

Reply:
Examples and details on how to deal with an application containing references is dealt with in the next section called "Update Application manual references". If there are references then this will involve migrating all the references over to the v2 syntax before the app can be released.

Reply:
Cool, I was wondering if those that do have references, will get confused if they are to do anything different.